app/services/scholars_archive/parent_query_service.rb

```diff
@@ -0,0 +1,7 @@
+module ScholarsArchive
+  class ParentQueryService
+    def self.query_parents_for_id(child_id)
```

Comment:
Can you write a test for this service and just mock the returns to guard against this changing unintentionally